tests/integration/test_backup_restore_new/test.py

```diff
@@ -192,6 +192,87 @@ def test_incremental_backup():
     assert instance.query("SELECT count(), sum(x) FROM test.table2") == "102\t5081\n"
 
 
+def test_increment_backup_without_changes():
+    backup_name = new_backup_name()
+    incremental_backup_name = new_backup_name()
+    create_and_fill_table(n=1)
+
+    system_backup_qry = "SELECT status, num_files, num_processed_files, processed_files_size, uncompressed_size, compressed_size, error FROM system.backups WHERE id='{id_backup}'"
+
+    assert instance.query("SELECT count(), sum(x) FROM test.table") == TSV([["1", "0"]])
+
+    # prepare first backup without base_backup
+    (id_backup, status) = instance.query(
+        f"BACKUP TABLE test.table TO {backup_name}"
+    ).split("\t")
+
+    (
+        backup_status,
+        num_files,
+        num_processed_files,
+        processed_files_size,
+        uncompressed_size,
+        compressed_size,
+        error,
+    ) = instance.query(system_backup_qry.format(id_backup=id_backup)).split("\t")
+
+    assert backup_status == "BACKUP_CREATED"
+    assert num_files == "11"
+    assert int(uncompressed_size) > 0
+    assert int(compressed_size) > 0
+    assert error == "\n"
+
+    # create second backup without changes based on the first one
+    (id_backup_snd, status_snd) = instance.query(
+        f"BACKUP TABLE test.table TO {incremental_backup_name} SETTINGS base_backup = {backup_name}"
+    ).split("\t")
+
+    (
+        backup_status_snd,
+        num_files_snd,
+        num_processed_files_snd,
```

Comment:
"snd"? What does it mean?

Reply:
It means second. But if you had the question, I reworded these variables in more friendly and meaningful way